frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -174,3 +174,42 @@ via `SERVER_FLAGS` object.
   be enabled or disabled separately.
 - Failure to resolve a code reference (unable to load module, missing module export etc.) will disable
   the plugin.
+
+### Component APIs
+
+There are currently two API packages exposed as part of the dynamic plugin SDK. The Stable API package and Internal API packages.
+Stable APIs have been vetted and are unlikely to change in terms of signature.
```

Comment:
Stable APIs cannot change without proper version update

Reply:
Removing doc for now. Will send a different PR which will generate docs based on the exposed APIs.